OptimizelySwiftSDK.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
   s.name                    = "OptimizelySwiftSDK"
   s.module_name	            = "Optimizely"
-  s.version                 = "3.4.0"
+  s.version                 = "3.5.0"
```

Comment:
This podspec version will be updated automatically by JC's script later. This manual fix here may cause the flow. You can remove this change for safety.

Reply:
This podspec version was not update to 3.4.1 at the last release since it was not released from master last time.